aptos-move/aptos-vm/src/aptos_vm.rs

```diff
@@ -2249,11 +2255,11 @@ impl AptosVM {
                     txn_data,
                     log_context,
                     traversal_context,
+                    self.is_simulation,
                 )?;
-                // Skip validation if this is part of tx simulation.
-                // This allows simulating multisig txs without having to first create the multisig
-                // tx.
-                if !self.is_simulation {
+                // Once "simulation_enhancement" is enabled, the simulation path also validates the
```

Comment:
Good comments, this and the code at 1146 make sense.

Reply:
Thanks!